### DISCUSSION TITLE
When running on Loihi2, WARNING: In the convolution layer of dimension (160, 128, 16) some convolution results might not be exact. Some convolution spikes originating from x_in=n*256-1 and its neighbours might get dropped.

### DISCUSSION THREAD

**Discussion by gwgknudayanga** [lava-nc/lava#899] [Q&A]
Hi,

* When i am running my convolutional neural network on Loihi2.
* I am getting following warning. What does that mean? how it is going to affect my results?
(My input resolution is 320x256 and 2 channels)

WARNING: In the convolution layer of dimension (160, 128, 16) some convolution results might not be exact. Some convolution spikes originating from x_in=n*256-1 and its neighbours might get dropped.

Thank you,

**Comment by Piotr45** [comment]
Hi @gwgknudayanga, did you manage to get the network working on Loihi? I'm currently trying to run a Conv model with an input size of 128x128x2, but it either fails to converge or hangs during the compilation process. I'd appreciate it if you could share your experience.

**Reply by gwgknudayanga** [reply]
Hi @Piotr45 , yes i am, i reduced the input resolution to 256x192x2 (For DVS data). And I make sure number of neurons in my network is less than 1M as i target single Loihi2 chip. The number of neurons is about 720000  in my case. 
* Are your data loader , loading numpy tensors instead of pytorch tensors? Because i can remember initially i was loading torch tensor initially and it was used to hang in that case.
